### ISSUE TITLE
AttributeError: /usr/lib/x86_64-linux-gnu/libseccomp.so.2: undefined symbol: seccomp_api_get

### ISSUE THREAD

**Issue by nomeata** [cptpcrd/pyseccomp#2]
I get this, trying your library on a Debian stable machine:
```
Traceback (most recent call last):
  File "sandbox/sandbox.py", line 4, in <module>
    import pyseccomp as seccomp
  File "/home/jojo/projekte/programming/pybotbrain/backend/env/lib/python3.7/site-packages/pyseccomp.py", line 121, in <module>
    _libseccomp.seccomp_api_get.argtypes = ()
  File "/usr/lib/python3.7/ctypes/__init__.py", line 369, in __getattr__
    func = self.__getitem__(name)
  File "/usr/lib/python3.7/ctypes/__init__.py", line 374, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
AttributeError: /usr/lib/x86_64-linux-gnu/libseccomp.so.2: undefined symbol: seccomp_api_get
```

**Comment by nomeata** [comment]
Guess Debian’s `libseccomp` is simply too old. It has 2.3.3-4, and some googling reveals
> The symbol in question is only available in libseccomp v2.4.0 and higher, so this is unsurprising...

**Comment by nomeata** [comment]
Maybe something like this would be a better idiom?
```diff
diff --git a/pyseccomp.py b/pyseccomp.py
index 4418cfd..74db6f5 100644
--- a/pyseccomp.py
+++ b/pyseccomp.py
@@ -118,11 +118,13 @@ _libseccomp.seccomp_arch_remove.restype = ctypes.c_int
 _libseccomp.seccomp_arch_native.argtypes = ()
 _libseccomp.seccomp_arch_native.restype = ctypes.c_uint32
 
-_libseccomp.seccomp_api_get.argtypes = ()
-_libseccomp.seccomp_api_get.restype = ctypes.c_uint
+if hasattr(_libseccomp, 'seccomp_api_get'):
+    _libseccomp.seccomp_api_get.argtypes = ()
+    _libseccomp.seccomp_api_get.restype = ctypes.c_uint
 
-_libseccomp.seccomp_api_set.argtypes = (ctypes.c_uint,)
-_libseccomp.seccomp_api_set.restype = ctypes.c_int
+if hasattr(_libseccomp, 'seccomp_api_set'):
+    _libseccomp.seccomp_api_set.argtypes = (ctypes.c_uint,)
+    _libseccomp.seccomp_api_set.restype = ctypes.c_int
 
 _libseccomp.seccomp_syscall_resolve_num_arch.argtypes = (ctypes.c_uint32, ctypes.c_int)
 _libseccomp.seccomp_syscall_resolve_num_arch.restype = ctypes.c_void_p
```

**Comment by cptpcrd** [comment]
> Guess Debian’s `libseccomp` is simply too old. It has 2.3.3-4, and some googling reveals
> 
> > The symbol in question is only available in libseccomp v2.4.0 and higher, so this is unsurprising...

Yes, that's the problem. I don't think I've ever actually tested this library on a system with libseccomp < 2.4.

> Maybe something like this would be a better idiom?
>
> ...

It's slightly more involved than that, because I'll want to make `get_api()` and `set_api()` handle errors nicely (like the interfaces to the libseccomp 2.5's notification API do), and then I'll need to update the tests to avoid using the APIs on libseccomp 2.3.

I'll put a fix for this together soon.
